user/db.go

```diff
@@ -73,161 +70,276 @@ func (s *Store) updateUser(ctx context.Context, user User) error {
 func (s *Store) getUser(ctx context.Context, user User) (User, error) {
 	u := User{}
 	err := s.db.GetContext(ctx, &u,
-		`SELECT user_id, username, nickname, first_name, last_name, email, last_login, salt, password, avatar, use_gravatar
+		`SELECT *
 		FROM people.users
-		WHERE (username = $1 AND username != '') OR (email = $2 AND email != '') OR user_id = $3
-		LIMIT 1;`, user.Username, user.Email, user.UserID)
+		WHERE (username = $1 AND username != '') OR (email = $2 AND email != '') OR (ldap_username = $3 AND ldap_username != '') OR user_id = $4
+		LIMIT 1;`, user.Username, user.Email, user.LDAPUsername, user.UserID)
 	if err != nil {
 		return u, fmt.Errorf("failed to get user from db: %w", err)
 	}
 	return u, nil
 }
 
-// getUsers will get a group of users
+// getUsers will get users
 func (s *Store) getUsers(ctx context.Context) ([]User, error) {
 	var u []User
 	err := s.db.SelectContext(ctx, &u,
-		`SELECT user_id, username, nickname, first_name, last_name, email, last_login, avatar, use_gravatar
+		`SELECT *
 		FROM people.users;`)
 	if err != nil {
 		return nil, err
 	}
 	return u, nil
 }
 
-// getUsersIDA will get a group of users by userId ASC
-func (s *Store) getUsersIDA(ctx context.Context) ([]User, error) {
+// getUsersSizePage will get users with page size
```

Comment:
Redundant comment, either add more explanation (eg what do `size` and `page` represent?) or remove

Reply:
Added comment